standup_notes/__main__.py

```diff
@@ -124,6 +164,36 @@ def iterate_weekday(day: date, func: Callable[[date], date]) -> date:
         return iterate_weekday(next_day, func)
 
 
+def delete_notes(amount):
+    file_to_delete = []
+    if len(str(amount)) != 8:
```

Comment:
What exactly is this check doing? Why 8?

Reply:
This checks if a valid date has been entered. Since the date entered has to be exactly 8 characters. I could add a clause to make sure only numbers are present

Reply:
Function to check dates has been added